spec.html

```diff
@@ -3900,6 +3899,11 @@ <h1>Well-Known Intrinsic Objects</h1>
     <h1>ECMAScript Specification Types</h1>
     <p>A specification type corresponds to meta-values that are used within algorithms to describe the semantics of ECMAScript language constructs and ECMAScript language types. The specification types include Reference, List, Completion Record, Property Descriptor, Environment Record, Abstract Closure, and Data Block. Specification type values are specification artefacts that do not necessarily correspond to any specific entity within an ECMAScript implementation. Specification type values may be used to describe intermediate results of ECMAScript expression evaluation but such values cannot be stored as properties of objects or values of ECMAScript language variables.</p>
 
+    <emu-clause id="sec-enum-specification-type">
+      <h1>The Enum Specification Type</h1>
```

Comment:
`Enum` is a type insofar as you can define it to be a collection of values. But the spec never uses it as a type (e.g., to specify the type of a parameter or a record field), and probably never will, because doing so would be less informative than listing the specific values allowed.

Reply:
Agreed. Still, the type is useful within this section to talk about how all enum values behave. Also we need a descriptive section header.